apps/beeai-sdk/pyproject.toml

```diff
@@ -24,32 +31,40 @@ build-backend = "uv_build"
 line-length = 120
 target-version = "py311"
 lint.select = [
-    "E",     # pycodestyle errors
-    "W",     # pycodestyle warnings
-    "F",     # pyflakes
-    "UP",    # pyupgrade
-    "I",     # isort
-    "B",     # bugbear
-    "N",     # pep8-naming
-    "C4",    # Comprehensions
-    "Q",     # Quotes
-    "SIM",   # Simplify
-    "RUF",   # Ruff
-    "TID",   # tidy-imports
+    "E", # pycodestyle errors
+    "W", # pycodestyle warnings
+    "F", # pyflakes
+    "UP", # pyupgrade
+    "I", # isort
+    "B", # bugbear
+    "N", # pep8-naming
+    "C4", # Comprehensions
+    "Q", # Quotes
+    "SIM", # Simplify
+    "RUF", # Ruff
+    "TID", # tidy-imports
     "ASYNC", # async
-# TODO: add "DTZ",   # DatetimeZ
-# TODO: add "ANN",   # annotations
+    # TODO: add "DTZ",   # DatetimeZ
+    # TODO: add "ANN",   # annotations
 ]
 lint.ignore = [
     "E501", # line lenght (annyoing)
     "N999", # invalid module name beeai-server (yeah, we use a dash, deal with it)
     "F403", # redundant with Pyright
 ]
 force-exclude = true
-exclude = ["tests"]
+
+[tool.pytest.ini_options]
+asyncio_mode = "auto"
 
 [dependency-groups]
 dev = [
     "pyright>=1.1.403",
+    "pytest>=8.4.1",
+    "pytest-asyncio>=1.1.0",
+    "pytest-httpx>=0.35.0",
     "ruff>=0.12.3",
 ]
```

Comment:
Bit unrelated but could you move this up, right below dependencies? It's now mixed down here among the cool configs